src/main/java/uk/ac/cam/cl/dtg/isaac/api/QuizFacade.java

```diff
@@ -1321,6 +1444,21 @@ private QuizAttemptDTO augmentAttempt(QuizAttemptDTO quizAttempt, @Nullable Quiz
         return augmentAttempt(quizAttempt, quizAssignment);
     }
 
+    private List<QuizUserFeedbackDTO> getUserFeedback(RegisteredUserDTO user, QuizAssignmentDTO assignment, IsaacQuizDTO quiz, List<RegisteredUserDTO> groupMembers) throws ContentManagerException, SegueDatabaseException {
```

Comment:
Could this be a manager method instead, so we keep non external methods out of the facade?